<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>PDS View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="current_obj_name">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>&lt;obj_name&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tab_display">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="label_tab">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <attribute name="title">
        <string>&amp;Label</string>
       </attribute>
       <layout class="QGridLayout" name="grid_layout_label">
        <item row="1" column="0">
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="three_d_table_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>&amp;3D Table</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4"/>
      </widget>
      <widget class="QWidget" name="table_tab">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <attribute name="title">
        <string>&amp;Table</string>
       </attribute>
       <layout class="QGridLayout" name="grid_layout_table">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableView" name="data_table_view">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rgb_image_tab">
       <attribute name="title">
        <string>RGB Image</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="image_tab">
       <attribute name="title">
        <string>&amp;Image</string>
       </attribute>
       <layout class="QGridLayout" name="image_grid_layout">
        <item row="1" column="0">
         <widget class="QWidget" name="image_view_widget" native="true">
          <layout class="QGridLayout" name="image_view_layout">
           <item row="0" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="image_toolbar_widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="export_button">
             <property name="toolTip">
              <string>Save Image As...</string>
             </property>
             <property name="text">
              <string>something</string>
             </property>
             <property name="icon">
              <iconset theme="image-x-generic">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Color Map</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="color_map_combo">
             <item>
              <property name="text">
               <string>gray</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Reds</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Greens</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Blues</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>afmhot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_heat</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cool</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>coolwarm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>jet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rainbow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hsv</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Paired</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Accent</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BrBG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BuGn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BuPu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CMRmap</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dark2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GnBu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Greys</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>OrRd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Oranges</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PRGn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PuOr</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PuRd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Purples</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RdBu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RdGy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RdPu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RdYlBu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>RdYlGn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spectral</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wistia</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YlGn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YlGnBu</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YlOrBr</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>YlOrRd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>autumn</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>binary</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bone</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>brg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bwr</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cividis</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>copper</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cubehelix</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>flag</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_earth</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_gray</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_ncar</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_rainbow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_stern</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gist_yarg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gnuplot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gnuplot2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>inferno</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>magma</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>nipy_spectral</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ocean</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pink</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>plasma</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>prism</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>seismic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>spring</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>summer</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>tab10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>tab20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>tab20b</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>tab20c</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>terrain</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>turbo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>twilight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>twilight_shifted</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>viridis</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>winter</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="summary_tab">
       <attribute name="title">
        <string>&amp;Summary</string>
       </attribute>
       <layout class="QGridLayout" name="grid_layout_summary">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>false</italic>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="summary_static">
             <property name="text">
              <string>Data structure summary for: </string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="current_summary_fpath">
             <property name="text">
              <string>&lt;file path&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="summary_table_view"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save_table_as"/>
    <addaction name="action_save_image_as"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save_table_as"/>
   <addaction name="action_save_image_as"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>../../view/ui</normaloff>../../view/ui</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_image_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="image-x-generic">
     <normaloff>../../view/ui</normaloff>../../view/ui</iconset>
   </property>
   <property name="text">
    <string>Save &amp;Image as...</string>
   </property>
   <property name="iconText">
    <string>Save Image as...</string>
   </property>
   <property name="toolTip">
    <string>Save Image as...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../view/ui</normaloff>../../view/ui</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_save_table_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="text-x-generic">
     <normaloff>../../view/ui</normaloff>../../view/ui</iconset>
   </property>
   <property name="text">
    <string>Save Table as...</string>
   </property>
   <property name="iconText">
    <string>Save Table as...</string>
   </property>
   <property name="toolTip">
    <string>Save Table as...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_select_pds_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>summary_table_view</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>_summary_table_row_clicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_save_table_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_export_csv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_save_image_as</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>_export_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tab_display</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_tab_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>export_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>_export_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>color_map_combo</sender>
   <signal>activated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>_color_map_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_select_pds_file()</slot>
  <slot>_quit()</slot>
  <slot>_summary_table_row_clicked(QModelIndex)</slot>
  <slot>_export_csv()</slot>
  <slot>_export_image()</slot>
  <slot>_tab_changed(int)</slot>
  <slot>_color_map_changed(QString)</slot>
 </slots>
</ui>
